notebooks/wine_predictor_analysis_report.ipynb

```diff
@@ -594,11 +612,41 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 19,
    "id": "06f40256-f308-4e7f-a3a7-8351360e8165",
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Best hyperparameters: {'svc__C': np.float64(1.934556578726365), 'svc__decision_function_shape': 'ovr', 'svc__gamma': 1, 'svc__kernel': 'rbf'}\n",
+      "Best cross-validation score: 0.6481740196078432\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Hyperparameter tuning for the best performing model\n",
+    "param_dist = {\n",
+    "    'svc__C': loguniform(1e-3, 1e3),\n",
+    "    'svc__kernel': ['linear', 'rbf', 'poly'],\n",
```

Comment:
Since we mentioned specifically SVM RBF in analysis, maybe we should not include this hyperparameter?